CITATION.cff

```diff
@@ -0,0 +1,20 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+type: "software"
+title: "mamba"
+doi: "10.5281/zenodo.7434205"
+date-released: "2019-03"
+authors:
+  - family-names: "QuantStack and Mamba Contributors"
+    given-names: ""
+    website: "https://github.com/mamba-org/mamba"
+repository-code: "https://github.com/mamba-org/mamba"
+url: "https://mamba.readthedocs.io/"
+abstract: "Mamba is a fast, robust, and cross-platform package manager. It is a drop-in replacement for Conda, with much faster dependency resolution and package installation."
```

Comment:
Is the "much faster dependency resolution" claim still true considering conda now uses libmamba under the hood?